api/challenge_api/app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
We won't be surfacing or returning any HTML through this API, so I think we're safe to remove this.